libs/accounts/recovery-phone/src/lib/recovery-phone.manager.in.spec.ts

```diff
@@ -11,6 +11,12 @@ describe('RecoveryPhoneManager', () => {
   let recoveryPhoneManager: RecoveryPhoneManager;
   let db: AccountDatabase;
 
+  const redisMock = {
```

Comment:
Super small nit, we've been prefix with mock. so mockRedis would be consistent perhaps.